pkg/proc/proc_test.go

```diff
@@ -4979,3 +4979,29 @@ func TestStepOutPreservesGoroutine(t *testing.T) {
 		}
 	})
 }
+
+func TestIssue2319(t *testing.T) {
+	// Check to make sure we don't crash on startup when the target is
+	// a binary with a mix of DWARF-5 C++ compilation units and
+	// DWARF-4 Go compilation units.
+
+	// Require CGO, since we need to use the external linker for this test.
+	protest.MustHaveCgo()
+
+	// The test fixture uses linux/amd64 assembly and a *.syso file
+	// that is linux/amd64, so skip for other architectures.
+	if runtime.GOOS != "linux" || runtime.GOARCH != "amd64" {
+		t.Skipf("skipping since not linux/amd64")
+	}
+	fixture := protest.BuildFixture("issue2319/", protest.BuildModeExternalLinker)
+
+	dest := "/tmp/issue2319.exe"
```

Comment:
Why is this needed? (also the call to MustHaveCgo needs to be `MustHaveCgo(t)`)

Reply:
Ack, sorry. Looks like I sent the wrong version. I'll fix that up.